src/interface.h

```diff
@@ -0,0 +1,15 @@
+#ifndef INTERFACE_H
+#define INTERFACE_H
+
+#include <iostream>
+#include <fstream>
+#include <libgcode/parser.h>
+
+class interface
+{
+public:
+    interface();
+    void read_gcode_file(std::string filename);
```

Comment:
The only effect of this function seems to be to print out the file. It is a useful function, but perhaps you could name it something more descriptive like `pretty_print_gcode_file`?

Reply:
Hi, i have attached a doxygen compatible comment above line 12. Hope this is ok for you.